src/main/features/claim/views/defendant-service-address.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import errorSummary, radioGroup, csrfProtection, saveAndContinueButton %}
+{% from "address.njk" import addressFields %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('Address for service') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form novalidate method="post">
+        {{ csrfProtection(csrf) }}
+
+        {% set errorType = form.errorFor('defendantsAddress') %}
+        <fieldset id="defendantsAddress[label]" class="form-group {% if errorType %} form-group-error {% endif %}">
```

Comment:
why don't you use `radioGroup` macro? Haven't you copied this macro from citizen fe?

Reply:
macros don't currently account for a pop out for each of the radio options and we can't get the desired effect using the macros